app/src/androidTest/java/com/github/geohunt/app/LeaderboardTest.kt

```diff
@@ -42,10 +43,15 @@ class LeaderboardTest {
         return object : User {
             override var displayName: String? = names[pos]
             override val uid: String = pos.toString()
-            override val profilePicture: LazyRef<Bitmap> = MockLazyRef("1") { throw NotImplementedError() }
+            override val profilePicture: LazyRef<Bitmap> = object : LazyRef<Bitmap> {
```

Comment:
It's generally a good practice to use uppercase letters for the first letter of each word in a constant's name, so you might consider renaming profilePicture to PROFILE_PICTURE.